app/javascript/leaflet/utils.js

```diff
@@ -0,0 +1,46 @@
+import { latLngBounds, Util, geoJSON } from "leaflet";
+
+export const bboxToBounds = (bbox) => {
+  bbox = bbox.split(" ");
+  if (bbox.length === 4) {
+    return latLngBounds([
+      [parseFloat(bbox[1]), parseFloat(bbox[0])],
+      [parseFloat(bbox[3]), parseFloat(bbox[2])],
+    ]);
+  } else {
+    throw new Error("Invalid bounding box string");
+  }
+};
+
+export const boundsToBbox = (bounds) => {
+  let sw = bounds.getSouthWest(),
+    ne = bounds.getNorthEast();
+
+  if (ne.lng - sw.lng >= 360) {
+    sw.lng = -180;
+    ne.lng = 180;
+  }
+  sw = sw.wrap();
+  ne = ne.wrap();
+  return [
+    Util.formatNum(sw.lng, 6),
+    Util.formatNum(sw.lat, 6),
+    Util.formatNum(ne.lng, 6),
+    Util.formatNum(ne.lat, 6),
+  ];
+};
+
```

Comment:
utils used to have a function called linkify. Not sure if linkify-html replaces it.